docs/新手指引/快速上手.md

```
@@ -0,0 +1,43 @@
+为了帮助开发者迅速掌握TinyEngine并深入理解其核心特性，我们设计了一个简单的实践上手案例：“开关按钮控制图片显示与隐藏”。
+这个案例直观展示了TinyEngine在交互设计，允许用户通过一个简单的开关按钮来控制图片的显示与隐藏状态。  
+
+通过这个案例，开发者可以快速学习如何利用TinyEngine的低代码特性，实现界面交互逻辑。
+
+### 效果图
+
+![](./imgs/showImg.png)
+
+### 第一步，新建页面
+
+![](./imgs/page1.png)
```

Markdownlint (0.35.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)